src/components/Message/hooks/useCreateMessageContext.ts

```diff
@@ -62,6 +66,7 @@ export const useCreateMessageContext = <
   }${message.updated_at}${message.deleted_at}${readBy}${message.status}${
     message.type
   }${message.text}${message.reply_count}`;
+  const membersValue = Object.keys(members).join;
```

Comment:
?
```suggestion
  const membersValue = Object.keys(members).join();
```

Reply:
3 am stuff, leme make sure it doesn't change anything